src/features/board/ui/PostCard.tsx

```diff
@@ -28,29 +69,37 @@ export default function PostCard({ post }: PostCardProps) {
             </p>
           </div>
         </div>
-        <button className="text-gray-400 hover:text-gray-600">
-          <EllipsisHorizontalIcon className="h-5 w-5" />
-        </button>
+        <div className="relative">
+          {isAuthor && (
+            <button
+              className=" text-gray-400 hover:text-gray-600"
+              onClick={() => setIsDropdownOpen(!isDropdownOpen)}
+            >
+              <EllipsisHorizontalIcon className="relative right-5 w-5" />
+            </button>
+          )}
+          <PostDropdown
+            isDropdownOpen={isDropdownOpen}
+            setIsEditing={setIsEditing}
+            handleDelete={handleDelete}
+          />
+        </div>
       </div>
 
       <div className="ml-15 pl-1">
-        <p className="text-gray-800 mb-3">{post.content}</p>
-        {post.imageUrl &&
-          post.imageUrl.map((url, index) => (
-            <div className="max-w-xs mb-3" key={url}>
-              <img
-                src={url}
-                alt={`게시물 이미지 ${index + 1}`}
-                className="rounded-lg border"
-              />
-            </div>
-          ))}
-        <div className="flex items-center gap-4 text-sm text-gray-500 mb-3">
+        <form className="text-gray-800 mb-3" onSubmit={handleSubmit}>
+          <Editor
+            onChange={setEditedContent}
+            isEditable={isEditing}
+            value={editedContent}
+          />
+        </form>
```

Comment:
<img width="884" height="539" alt="Image" src="https://github.com/user-attachments/assets/55de4042-53ba-4dbb-978f-3a621dd7e80b" />

```suggestion
        </form>

        {post.imageUrl && (
          <div className="mb-3 px-5">
            {post.imageUrl.map((url, idx) => (
              <img
                key={idx}
                src={url}
                alt={`post-image-${idx}`}
                className="rounded-lg mb-2 max-h-96 object-contain"
              />
            ))}
          </div>
        )}

```

이미지 확인차 코드를 이렇게 추가해봤는데 잘 나오네요! 기능 구현때 진행하려고 빼두신 건가요??

Reply:
글
이미지
글
이런식으로 있을때 순서같은게 보장되나요?
html형식으로 백엔드에 저장 후 이미지만 외부 s3같은곳에 저장해서 에디터에서 바로 html 파싱해서 랜더링하려구 했습니다

Reply:
아 글과 사진을 따로 렌더링 하시는 줄 알았네요! 해당 방식이라면 상관없을 것 같습니다~